src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css

```diff
@@ -694,6 +694,31 @@ However we always want it to be the width of the container it is resizing.
 	border-bottom: 1px solid var(--separator-border);
 }
 
+/* views created by the action-bar actions */
```

Comment:
Are these css duplicated from the previous control? If yes, can you please add the class names to the older css 

for ex:

class1 {
 css
}

becomes 

class1,
class2 {
css code
}